packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.CustomMenuList.Example.tsx

```diff
@@ -1,12 +1,79 @@
 import * as React from 'react';
+import { useConstCallback } from '@uifabric/react-hooks';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
-import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
+import { ISearchBoxStyles, SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
-import './ContextualMenuExample.scss';
 import { IContextualMenuListProps, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
 import { IRenderFunction } from 'office-ui-fabric-react/lib/Utilities';
 
-const ITEMS: IContextualMenuItem[] = [
+const wrapperStyle: React.CSSProperties = { borderBottom: '1px solid #ccc' };
```

Comment:
Most often what we've done for these styles instead of `React.CSSProperties` is `mergeStyleSets` with a class name for each thing, which I think was to try and discourage people from using the native `style` prop? But I'm also not sure how solid the reasoning behind that is. (I've also started usually putting styles below the main example since they're usually not the main point of the example.)

Reply:
Huh, weird, I've usually heard that we shouldn't use `mergeStyleSets` for menial things like this because it contributes to perf hits. I'll move the styles below the example though.

Reply:
Interesting, I'd be curious about more background on that. I'm guessing that applies mainly when it's done within a component's render method? Though if we're trying to model best practices, maybe we shouldn't be showing something similar to a bad practice in examples...the trouble is that using what I *think* is the current best practice (though now about to be deprecated...) of styled components makes examples way too complicated.

Reply:
Yeah, might be good knowing what @dzearing things about this, since "best practice" is very subjective.